conda/install.py

```diff
@@ -618,10 +619,20 @@ def link(pkgs_dir, prefix, dist, linktype=LINK_HARD, index=None):
         sys.exit('Error: pre-link failed: %s' % dist)
 
     info_dir = join(source_dir, 'info')
+
+    if not os.path.isfile(join(info_dir, "files")):
+        error_and_exit("Installing %s requires a minimum conda version of 4.3." % dist)
```

Comment:
I need to wait on merging this PR just to make sure we get the `files.psv` stuff resolved in the meeting on Friday.